docs/user/fitting/fitting.md

```diff
@@ -212,6 +212,57 @@ complete_flow = CompleteDFTvsMLBenchmarkWorkflow(
     ...
 )
 ```
+### Finetuning MACE-MP-0
+
+It is also possible to finetune MACE-MP-0. To do so, you need to install MACE-torch 0.3.7. 
+Currently, this can only be done by cloning the git-repo and installing it from there: []()
+
+It is now important that you switch off the default settings for the fitting procedure (use_defaults_fitting=False).
+If one wants to perform very low-data fine-tuning (less than 4 additional structures), please also think about switching off `preprocessing_data` 
```

Comment:
@QuantumChemist can you confirm that this is enough to skip the train, test split and simply take all data?

Reply:
But then the MLIPMaker will only run if train.extxyz and test.extxyz files are present in the database_dir
https://github.com/autoatml/autoplex/blob/fdbee8ec41360773ac26fead37e437dc39586180/autoplex/fitting/common/flows.py#L182

Reply:
Yeah. I will only add the option not change the default. This is what I have done for the rest of the implementations here as well. The defaults have not changed.